README.md

```diff
@@ -1,12 +1,5 @@
-# lambda-template
-Template for creating lambda repositories
-A template for quickly starting a new AWS lambda project.
-
-## Naming
-Naming conventions:
-* for a vanilla Lambda: `lambda-<context>`
-* for a Cloudformation Transform macro: `cfn-transform-<context>`
-* for a Cloudformation Custom Resource: `cfn-cr-<context>`
+# lambda-budgets
+Lambda that runs on a cron to create AWS Budgets.
```

Comment:
what does it mean to create a budget?  is it creating a report or a quota?

Reply:
Neither or both -- it is creating an [AWS Budget](https://aws.amazon.com/aws-cost-management/aws-budgets/)